indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -233,6 +245,12 @@ public TaskStatus call()
                               command.add(String.format("-Ddruid.host=%s", childHost));
                               command.add(String.format("-Ddruid.port=%d", childPort));
 
+                              if(config.isSeparateIngestionEndPoint()) {
+                                command.add(String.format("-Ddruid.indexer.task.chathandler.service=%s", "druid/chathandler"));
```

Comment:
actually I don't think I can change it somehow so that it does not register as valid ZKPath as *if* the peon were to somehow use this `serviceName`, it will just take the String as it is and try to announce it. What can be done here as you suggested to change the name to something like `placeholderServiceName` and add a comment form where the actual `serviceName` will be taken ?